examples/js/nextjs-new/README.md

```diff
@@ -0,0 +1,53 @@
+|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| /api/test1    | API has a top-of-module Promise that rejects, but its result is not awaited.                                                                                                   |
+| /api/test2    | API has a top-of-module exception.                                                                                                                                             |
+| /api/test3    | API has has an exception in its request handler.                                                                                                                               |
+| /api/test4    | API uses a try/catch to handle an exception and records it.                                                                                                                    |
+| /ssr/test1    | getServerSideProps throws an Error.                                                                                                                                            |
+| /ssr/test2    | getServerSideProps returns a Promise that rejects.                                                                                                                             |
+| /ssr/test3    | getServerSideProps calls a Promise that rejects, but does not handle the rejection or await its result (returning synchronously).                                              |
+| /ssr/test4    | getServerSideProps manually captures an exception from a try/catch.                                                                                                            |
+| /client/test1 | There is a top-of-module Promise that rejects, but its result is not awaited.                                                                                                  |
+| /client/test2 | There is a top-of-module exception. _error.js should render.                                                                                                                   |
+| /client/test3 | There is an exception during React lifecycle that is caught by Next.js's React Error Boundary. In this case, when the component mounts. This should cause _error.js to render. |
+| /client/test4 | There is an unhandled Promise rejection during React lifecycle. In this case, when the component mounts.                                                                       |
+| /client/test5 | An Error is thrown from an event handler.                                                                                                                                      |
+
+## Known Issues
+
+## Errors are not captured in some scenarios
+
+With a local production build (`npm run build && npm start`):
```

Comment:
I'm slightly concerned about the ambiguity of saying the test fails.  What we really mean is that the error is not reported in this scenario and we know that because the test fails.  Can we perhaps say e.g. "API scenario X doesn't work" rather than "API test X"?  And perhaps rename the tests accordingly?

If you think I'm being overly fussy let me know but I can imagine coming to this and thinking "ok the test fails, but is that an issue with the test or does the functionality actually not work".

Reply:
Yeah I think scenario is a better word 👍